src/components/common/data-row/data-row.scss

```diff
@@ -0,0 +1,5 @@
+.data-row-exclamation-indicator {
```

Comment:
as per previous comment. why the new file?

Reply:
moved styling into typography.scss